packages/minecraft/src/Difficulty.ts

```diff
@@ -0,0 +1,8 @@
+export enum Difficulty {
+    PEACEFUL,
+    EASY,
+    NORMAL,
+    HARD,
+    COUNT,
```

Comment:
Is `COUNT` intentional?

Reply:
Yes, everything that you find in `minecraft` package is the same as is in the game. (So it can be used for more purposes than only in JSP).
(Source: https://mojang.github.io/bedrock-protocol-docs/html/enums.html)